pandas/tseries/tests/test_base.py

```diff
@@ -861,21 +861,26 @@ def test_nat(self):
         self.assertIs(pd.DatetimeIndex([])._na_value, pd.NaT)
 
         for tz in [None, 'US/Eastern', 'UTC']:
+
             idx = pd.DatetimeIndex(['2011-01-01', '2011-01-02'], tz=tz)
             self.assertTrue(idx._can_hold_na)
 
             tm.assert_numpy_array_equal(idx._isnan, np.array([False, False]))
             self.assertFalse(idx.hasnans)
-            tm.assert_numpy_array_equal(idx._nan_idxs,
-                                        np.array([], dtype=np.int64))
+
```

Comment:
pls don't change the formatting unless it's germane (this line is too long anyhow)